.github/workflows/devnet_deploy_contracts.yml

```
@@ -0,0 +1,30 @@
+name: Deploy devnet contracts
```

```suggestion
name: Deploy Devnet contracts
```

To match the naming of the other actions.